vendor/bat-native-ledger/include/bat/ledger/ledger.h

```diff
@@ -50,6 +50,8 @@ using GetAddressesCallback =
 using HasSufficientBalanceToReconcileCallback = std::function<void(bool)>;
 using FetchBalanceCallback = std::function<void(ledger::Result,
                                                 ledger::BalancePtr)>;
+using GetGrantCaptchaCallback = std::function<void(std::string,
```

Comment:
Same comment as above: let's use `const std::string&` for these parameters if possible.

Reply:
fixed